personal_assistant/contacts/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+from . import views
+
+app_name = 'contacts'
+
+urlpatterns = [
+    path('contacts/', views.main, name='main'),
+    path('contacts/add_contact/', views.add_contact, name='add_contact'),
+    path('contacts/detail/<int:contact_id>', views.detail, name='detail'),
+    path('delete/<int:contact_id>', views.delete_contact, name='delete'),
+]
```

Comment:
Так як тут лише CRUD-операції і загальна сторінка для контактів то можна дещо спростити назви:
```suggestion

from .views import create, delete, main, read

app_name = 'pa_contact'

urlpatterns = [
    path('', main, name='main'),
    path('add/', create, name='create'),
    path('<int:contact_id>/', read, name='read'),
    path('<int:contact_id>/delete/', delete, name='delete'),
]
```